addons/common/XEH_preInit.sqf

```diff
@@ -78,3 +79,19 @@ activateAddons GVAR(addons);
 ["CAManBase", "InitPost", CBA_fnc_randomizeFacewear] call CBA_fnc_addClassEventHandler;
 
 ADDON = true;
+
+["CBA QTE", "QTE_Up_Key", ["↑", "Up key used in QTE events."], {}, {
```

Comment:
Personally have a problem with `QTE events` cuz it expands into `Quick-Time Event events`.

Making it `Quick-Time Events` ain't just 2 letters longer, I admit, but shouldn't matter:
```
Up key used in QTE events.
Up key used in Quick-Time Events.
```